native/cocos/renderer/gfx-wgpu/WGPUTexture.cpp

```diff
@@ -99,6 +105,7 @@ void CCWGPUTexture::doInit(const TextureViewInfo &info) {
 
 void CCWGPUTexture::doInit(const SwapchainTextureInfo &info) {
     if (_swapchain) {
+        printf("swapchain %d, %d\n", info.width, info.height);
```

Comment:
Why not to use `CC_LOG_DEBUG` ?

Reply:
this is a "issue" using with emscripten. When using CC_LOG_XXXX/std::cout..., not sure why some function like `get_local_time` has been called, and those functions doesn't work happy with emscripten, maybe more flags are needed to export system funcs. While printf needs nothing.

Reply:
Can modify CC_LOG_DEBUG to adapt wgpu.